helm-v3/005-plugins.md

```diff
@@ -63,8 +68,28 @@ used for executing Lua plugins mirrors the pattern for writing command-line
 applications. A `main()` function will be executed with access to STDIN, STDOUT,
 STDERR, and all command-line parameters and options.
 
-### Lua Plugin Sandboxing
+#### Lua Plugin Sandboxing
 
 Like charts, each plugin will be executed in an isolated Lua sandbox. Unlike
 chart scripts, plugins will be granted access to all core libraries, and will
 not require security prompts to accept.
+
+## Installing Plugins
+A single plugin can be installed via `helm plugin install`.
+
+A set of plugins can also be defined in a yaml file and installed during the `helm init` process by using a flag called `--custom plugins <file.yaml>`
+where `file.yaml` looks like this:
+```
+plugins:
+- name: helm-template
+  version: 0.1.0
```

Comment:
How will we resolve versions?

Reply:
@michelleN we discussed this in the developer call. Figuring out how to handle fetching the right version when the tip of master isn't the same as what was declared is difficult.

What about dropping the version info in the first go at this? This was suggested by @technosophos